crates/locate-bin/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "locate-bin"
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+
+[dependencies]
+which = "4.4.0"
```

Comment:
If you're looking for the binary on the path, is it necessary to know it's absolute path? For example, we have otherwise just built a command around the bare command and let std::process::Command figure it out, since it will do its own resolution using $PATH etc.

Reply:
@jgraettinger when mounting the flow-connector-init binary on the connector we need to know the absolute path to be able to mount it, hence the use of `which`